src/transformers/models/llama/configuration_llama.py

```diff
@@ -187,6 +191,7 @@ def __init__(
         self.attention_bias = attention_bias
         self.attention_dropout = attention_dropout
         self.mlp_bias = mlp_bias
+        self.head_dim = head_dim
```

Comment:
what you can do here is if head_dim is None: `self.head_dim = self.hidden_size // self.num_heads`

Reply:
Thanks for the suggestion! Added.